_layouts/docwithnav.html

```diff
@@ -16,6 +16,7 @@
 	<div class="row">
 		<!-- Navigation menu -->
 		<div class="col-md-3 hidden-sm hidden-xs" id="wrapper">
+			{% include docversionselector.html %}
```

Comment:
@bgrant0607 @nikhiljindal @janetkuo, maybe we can merge this dropdown menu without other changes in this PR, so the sitemap tool will include v1.1 and users are still guided to v1.0 by default?